stable_baselines/common/cmd_util.py

```diff
@@ -15,8 +15,60 @@
 from stable_baselines.common.vec_env import DummyVecEnv, SubprocVecEnv
 
 
+def make_vec_env(env_id, n_envs=1, seed=None, start_index=0,
+                 use_subprocess=False, start_method=None,
+                 monitor_path=None, wrapper_class=None, env_kwargs=None):
+    """
+    Create a wrapped, monitored VecEnv.
+
+    :param env_id: (str or gym.Env) the environment ID or the environment class
+    :param n_envs: (int) the number of environment you wish to have in parallel
+    :param seed: (int) the inital seed for random number generator
+    :param start_index: (int) start rank index
+    :param start_method: (str) method used to start the subprocesses.
+        See SubprocVecEnv doc for more information
+    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or `DummyVecEnv` when
+        `n_envs` > 1, `DummyVecEnv` is usually faster. Default: False
+    :param wrapper: (gym.Wrapper) Additional wrapper to use on the environment
```

Comment:
Docs for `monitor_path`. Docs say `wrapper` when parameters say `wrapper_class`.

Reply:
nice catch ;)